code/src/pgsql_codec/pgsql_codec.ml

```diff
@@ -279,6 +279,13 @@ module Decode = struct
 
   let create () = { buf = Buffer.create 4096; needed_bytes = -1 }
 
+  let rec string_list () =
```

Comment:
I know I told you to put this here when we were pairing, but I think now that it's settled this function should be moved closer to the other reader functions (so near line 69)

Reply:
@orbitz this was the only function I wasn't able to refactor, I got me weird build issues once I moved it to Reader.

Reply:
fixed